shimmingtoolbox/prepare_fieldmap.py

```diff
@@ -81,4 +84,9 @@ def prepare_fieldmap(phase, echo_times, affine, unwrapper='prelude', mag=None, m
     fieldmap_rad = phasediff_unwrapped / echo_time_diff  # [rad / s]
     fieldmap_hz = fieldmap_rad / (2 * math.pi)  # [Hz]
 
+    # Gaussian blur the fieldmap
+    if gaussian_filter == True:
```

Comment:
```suggestion
    if gaussian_filter:
```
This can be simplified since the default value will be `False`